src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+import './_checkbox.scss';
```

Comment:
Is it the same as in registration page ?

Reply:
Yes